conferences/2018/css.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "CSSDay",
+    "url": "http://2018.cssday.it/",
+    "startDate": "2018-03-16",
+    "endDate": "2018-03-16",
+    "city": "Faenza",
+    "country": "Italy",
+    "twitter": "@cssdayit",
```

Comment:
Thanks for the submission. There is an extra `,` here, the test should've caught it! I'll add it myself in the meanwhile. 👍

Reply:
Oh good catch! Missed that when deleting not needed lines.